### DISCUSSION TITLE
Is it possible to add multiple columns assigned as "numbers"in collections?

### DISCUSSION THREAD

**Discussion by FSNaval** [benjaminjonard/koillection#507] [Ideas]
Good day all,

I am using coillection to catalogue my vinyl albums collection and i would like to know how much money i have spent so far.

For each vinyl album, i am assigned a new field with "price" and the value is "number". This number is the cost of the album.

Can it be made possible that koillection adds all the fields with "value" number in a collection, so as to give a total sum of these numbers?

**Comment by benjaminjonard** [comment]
Hello,

This is something I will be working on in the near future (someone also asked for something similar here #502)
I'm planning on adding a new type of field called "Price",  and display somewhere the sum of all "Price" fields with the same label somewhere on the collection page.

I'm just not sure how I'm going to handle nested collections for now
